posix.mak

```diff
@@ -380,54 +385,69 @@ $(DOC_OUTPUT_DIR)/dmd-%.html : %.ddoc dcompiler.dd $(DDOC) $(DMD)
 $(DOC_OUTPUT_DIR)/dmd-%.verbatim : %.ddoc dcompiler.dd verbatim.ddoc $(DMD)
 	$(DMD) -c -o- -Df$@ verbatim.ddoc dcompiler.dd $<
 
+$(DOC_OUTPUT_DIR):
+	mkdir -p $@
```

Comment:
When I locally ran just `pdf`, the `web` folder was never created and thus the targets failed. Hence I added `web` as order-only dependency to the respective targets.